final_prep/annotations.rb

```diff
@@ -16,28 +21,35 @@ def build_a_bear(name, age, fur, clothes, special_power)
     'sayings' => [greeting, power_saying, "Goodnight my friend!"],
     'is_cuddly' => true,
   }
+  # Create a method to return the details of your built_bear hash as a return statement
   return built_bear
 end
-
+  # Create two new instances of teh build_a_bear method
```

Comment:
For this, I wouldn't use the term "instance" - I'd save that for when you are creating an object from a Class which is that special type of method. 

You can instead say, "call the build_a_bear method" or "invoke the build_a_bear method"!

Reply:
Ah yes, good call!